convert.pl

```diff
@@ -29,16 +30,18 @@ sub uint_from_bits
     return 'uint16_t' if ($bits <= 16);
     return 'uint32_t' if ($bits <= 32);
     return 'uint64_t' if ($bits <= 64);
-    die 'Value too large';
+    confess 'Value too large for bits: '. $bits;
```

Comment:
I'm not quite sure why you've switched from die to confess here as the function is only called in one place?

Reply:
I hit a field that was 80 bit wide, so that it broke and I just wanted to see where it came from. Leaving it with a simple die is ok, I just did not read the code thoroughly enough.